### PR TITLE
[Feature] docker run action

### PR DESCRIPTION
**Pull request by Dexus**
**What**:

Have our own docker-run action to be sure its save!

**Why**:

Secure Code

**How**:

**Checklist**:

- [x] Tests
- [ ] CHANGELOG.md entry
- [x] Documentation

### CODE REVIEW

**Comment by Dexus** [comment]
rebased onto current main branch

File: docker-run/RELEASES.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### 4.x.x
+
+- Added a single output to the `action.yml`. This enables you to capture output from the container
+  using `::set-output name=container_output::somethinggoeshere` (Added
+  by [Trevor Sullivan](https://github.com/pcgeek86))
+- Added support for WORKDIR and ENV (Added by [Linas Daneliukas](https://github.com/LDaneliukas))
+
+### 3.0.0
+
+- Upgrade to docker 20.10 https://github.com/addnab/docker-run-action/pull/12
```
Comment (y0urself) [inline docker-run/RELEASES.md:12]:
```suggestion
- Upgrade to docker 20.10 [#12](https://github.com/addnab/docker-run-action/pull/12)
```

File: docker-run/githistory.log [hunk 2]
```diff
@@ -0,0 +1,76 @@
+*   11425ee : Merge remote-tracking branch 'pcgeek86/main' into main
```
Comment (y0urself) [inline docker-run/githistory.log:1]: This file can be removed I guess.

**Review by y0urself — CHANGES_REQUESTED** [review]
Can you also please give details where we want to use this action?

**Comment by Dexus** [comment]
Final version, now ready to review. 🗡️  

If you have better solutions to get the ENV with correct escape let me know (https://github.com/greenbone/actions/pull/25/files#diff-4823a7fc703bee3ba23e311d17633b03112726b6f25bde2f53c8556a4de7a988R3-R11).

File: docker-run/githistory.log (hunk 2 above)
Comment (Dexus) [reply docker-run/githistory.log:1]: This file is there to keep the changes that are made, before moved to our repo without any history. I have no clue how we handle source we copy and modify - only link and LICENSE copy?

File: docker-run/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Abdud Dayan Adeeb
```
Comment (bjoernricks) [inline docker-run/LICENSE:3]: If the action code got changed after the forking we should add our copyright here too
```suggestion
Copyright (c) 2020 Abdud Dayan Adeeb
Copyright (c) 2021 Greenbone Networks GmbH
```

File: docker-run/RELEASES.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# addnab/docker-run-action Releases
```
Comment (bjoernricks) [inline docker-run/RELEASES.md:1]: I don't think we should keep this file.

**Review by bjoernricks — CHANGES_REQUESTED** [review]

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**